app/src/app/api/v0/city/[city]/population/route.ts

```diff
@@ -0,0 +1,24 @@
+import { db } from "@/models";
+import { apiHandler } from "@/util/api";
+import { createCityRequest, createPopulationRequest } from "@/util/validation";
+import createHttpError from "http-errors";
+import { Session } from "next-auth";
+import { NextResponse } from "next/server";
+import { randomUUID } from "node:crypto";
+
+export const POST = apiHandler(
+  async (
+    req: Request,
+    context: { session?: Session; params: Record<string, string> },
+  ) => {
+    const body = createPopulationRequest.parse(await req.json());
+    if (!context.session) {
+      throw new createHttpError.Unauthorized("Unauthorized");
+    }
```

Comment:
Later on we will have to do access control here as well, so a user can't modify the Population entries of a city that he doesn't "own"/ have access to in our database.